interface/patient_file/report/patient_report.php

```diff
@@ -354,7 +354,7 @@ function show_date_fun(){
         // trim to a reasonable length for display purposes --cfapress
         $maxReasonLength = 20;
         if (strlen($result["reason"]) > $maxReasonLength) {
-            $result['reason'] = substr($result['reason'], 0, $maxReasonLength) . " ... ";
+            $result['reason'] = mb_substr($result['reason'], 0, $maxReasonLength) . " ... ";
```

Comment:
Lets place a comment at top of this line:
``````
// The default encoding for this mb_substr() call is set near top of globals.php
``````
(just to have as a reminder if add more of these functions in the future)